hpeOneView/resources/settings/appliance_device_snmp_v3_users.py

```diff
@@ -41,10 +41,10 @@ class ApplianceDeviceSNMPv3Users(object):
     """
     URI = '/rest/appliance/snmpv3-trap-forwarding/users'
 
-    def __init__(self, con):
-        self._client = ResourceClient(con, self.URI)
+    def __init__(self, connection, data=None):
+        super(ApplianceDeviceSNMPv3Users, self).__init__(connection, data)
 
-    def create(self, resource, timeout=-1):
+    def create(self, data, timeout=-1):
```

Comment:
no need to add create method explicitly since it is already part of parent class. Remove that

Reply:
if you want to add any extra lines of code to that method, then only you need to declare that method explicitly.